packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -52,6 +52,8 @@ async function main() {
         .option("-v, --verbose", "Enables verbose logging")
         .option("-b, --browserAuth", "Enables browser auth which may require a user to open a url in a browser.")
         .option("-m, --enableMetrics", "Enable capturing client & ops metrics")
+        .option("-createId", "--createTestId", "Flag indicating whether to create a document corresponding \
```

Comment:
Thanks, I'll remove the single-dash version!

But I think it makes sense to keep the flag name as `--createTestId` and not `--createIfNotExists` because if set to true, it does create the document regardless of whether it previously existed.